packages/app/src/features/liquidity-flow/components/borrow-decrease-modal/borrow-decrease-modal.tsx

```diff
@@ -64,7 +64,7 @@ export function BorrowDecreaseModal({
         title="How much to pay off"
         buttonProps={{
           label: `Pay off ${value} ${tokenSymbol}`,
-          onClick: () => onSend({ value: BigInt(value), token }),
+          onClick: () => onSend({ value: BigInt(value), token: tokenName }),
```

Comment:
It seems we can update `onSent` to take `tokenName` as well.

Reply:
Updated

Reply:
Thank you.